src/ui/components/todomvc-app/component.ts

```diff
@@ -1,24 +1,22 @@
 import Component, { tracked } from "@glimmer/component";
 import Navigo from 'navigo';
 import TodoStore from '../../../utils/todo-store';
-import Todo from '../utils/todo';
+import Todo from '../../../utils/todo';
+import { ENTER } from '../../../utils/keys';
 
 const router = new Navigo(null, true);
 
-
-
 export default class TodoMVCApp extends Component {
-
-  storage: TodoStore;
+  todoStore = new TodoStore();
 
   @tracked todos: Todo[] = [];
```

Comment:
In plain JavaScript the above is also true ([see here](https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015-loose%2Ces2016%2Ces2017%2Cstage-2&targets=&browsers=&builtIns=false&code=class%20Bar%20%7B%20%7D%0D%0A%0D%0Aclass%20Foo%20%7B%0D%0A%20%20%20%20blah%20%3D%20%5B%5D%0D%0A%7D&experimental=true&loose=true&spec=false&playground=true)):

```js
class Bar { }

class Foo {
    blah = []
}
```

Output:

```js
"use strict";

function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }

var Bar = function Bar() {
    _classCallCheck(this, Bar);
};

var Foo = function Foo() {
    _classCallCheck(this, Foo);

    this.blah = [];
};
```